lib/cli/command.ts

```diff
@@ -46,6 +46,9 @@ export const loadXudClient = async (argv: Arguments<any>) => {
   await loadXudConfig(argv);
 
   const certPath = argv.tlscertpath || path.join(argv.xudir, 'tls.cert');
+  if (!fs.existsSync(certPath)) {
```

Comment:
Rather than do an `existsSync` call up front. We should wrap the `readFileSync` call and then handle the `ENOENT` error with a custom error message, otherwise rethrow the error.